docs/http-challenge/azbs.md

```diff
@@ -0,0 +1,94 @@
+# Variables for Azure blob storage http-challenge
+
+| Variable              | Required | Default   | Description
+|-----------------------|----------|-----------|------------
+| acme_azbs_resource_group   | yes      |           | Name of the Azure resource group to which the storage account has been allocated
+| acme_azbs_storage_account_name    | yes      |           | Azure storage account name which should be used
+| acme_azbs_container_name    | yes      |           | Azure container name which will be used/created in Azure storage account
+| acme_azbs_subscription_id | yes       | | Azure subscription id
+| acme_azbs_client_id | yes | | Client ID of service principal/application
+| acme_azbs_secret | yes | | Value of secret of service principal/application (Note: not the ID)
+| acme_azbs_tenant_id | yes | | Tenant ID of service principal/application
+
+## Validation
+
+You have to create a service principal/application in the Azure Active Directory.
```

Comment:
I don't think that we should document how to create a SP. We should just link to the azure-docs. Maybe to this: https://learn.microsoft.com/en-us/azure/active-directory/develop/app-objects-and-service-principals

Reply:
You are right. I will just link to the azure-docs